pylsp/config/config.py

```diff
@@ -50,7 +50,7 @@ def __init__(self, root_uri, init_opts, process_id, capabilities):
         for entry_point in pkg_resources.iter_entry_points(PYLSP):
             try:
                 entry_point.load()
-            except ImportError as e:
+            except Exception as e:
```

Comment:
```suggestion
            except Exception as e:  # pylint: disable=broad-except
```

I forgot about this, which will solve the error reported by Pylint